meshroom/core/graph.py

```diff
@@ -452,6 +453,10 @@ def addNewNode(self, nodeType, name=None, position=None, **kwargs):
         Returns:
              The newly created node.
         """
+        # Apply default attribute value overrides set in preferences 
+        for a in Preferences().getAttributeOverrides(nodeType):
+            kwargs[a._name] = a._value
```

Comment:
Would be better to deal with in the generic NodeDesc where the actual node default value is stored.
We should keep the original value in memory, but the override would be better in the same place.